docs/pages/server-access/getting-started.mdx

```diff
@@ -0,0 +1,384 @@
+
+   iii. Create a *join token* to link `ssh.example.com` with the `auth.example.com` cluster:
+
+   - `sudo tctl tokens add --type=node`  
+   - "Node" mode specifies that the Teleport Node that will join will act as an SSH server.  
+
+   iv. You should see:
+
+   ```bash
+   # Join cluster
+   sudo teleport start \
+     --roles=node \
+     --token=d62629ea83e181644e0778af22ec3a8e \
+     --ca-pin=sha256:c75f8be5b21a5cc1b6f91a71200f30a9a083dfea4fc47ec1c55143ccce8882e4 \
+     --auth-server=172.31.17.124:3025
+   ```
+
+   <Admonition type="warning" title="Warning">
+     Remember to replace the IP address supplied by default above with the `auth.example.com` proxy address: `auth.example.com:443`.
+   </Admonition>
+
+2. On your second terminal, `ssh.example.com`:
+
+   i. Run the command above with the correct `auth.example.com` public proxy address on the machine you wish to connect.
+
+   ii. You should now be able to view both Nodes at: `https://auth.example.com/web/cluster/auth.example.com/nodes` in the Web Interface after logging in with your the user `auth-admin`.
+
+   ![Both Nodes in](../../img/server-access/teleport_ui.png)
+
+## Step 2a. Use tsh to connect to the cluster
+
+Now, we've got our cluster up and running. Let's see how easy it is to connect to our two nodes:
+
+1. On your local machine, log in through `tsh`: 
+
+   i. `tsh login --proxy=auth.example.com:443 --user=auth-admin`  
+   ii. You'll be prompted to supply the password and One Time Passcode we set up previously.
+
+2. `auth-admin` will now see:
+
+   ```bash
+   Profile URL:        https://auth.example.com:443
+     Logged in as:       auth-admin
+     Cluster:            auth.example.com
+     Roles:              access, admin, editor
+     Logins:             root, ubuntu, ec2-user
+     Kubernetes:         disabled
+     Valid until:        2021-04-30 06:39:13 -0500 CDT [valid for 12h0m0s]
+     Extensions:         permit-agent-forwarding, permit-port-forwarding, permit-pty
+   ```
+
+  - `auth-admin` is now logged into the `auth.example.com` cluster and `auth` Node via Teleport SSH - no third-party tool required.
+
+3. `auth-admin` can now execute the following to find the cluster's `nodenames` which are used for direct SSH connections:
+
+   i. `tctl nodes ls`  
+   ii. The `auth` node is located on the bottom line below:
+
+   ```bash
+   Nodename         UUID                                 Address        Labels                                
+   ---------------- ------------------------------------ -------------- ------------------------------------- 
+   ip-172-31-30-155 8f1e2719-3161-400f-b664-3279cf046511                env=example,hostname=ip-172-31-30-155 
+   ip-172-31-17-124 e7719967-b6eb-4fc0-8e53-00fac6d896d4 127.0.0.1:3022 env=example,hostname=ip-172-31-17-124 
+   ```
+
+4. Using the command `tsh ssh root@ip-172-31-17-124` locally, `auth-admin` can SSH into the `auth` Node and do the following:
```

Comment:
I would add a tab switcher here that will use `tsh` by default but also show a bit more lengthy `ssh` command equivalent:

`ssh -J tele.example.com user@host`

This will answer a very common first question of whether users can use ssh to connect to nodes.